internal/services/network/public_ip_prefix_resource.go

```diff
@@ -63,6 +63,17 @@ func resourcePublicIpPrefix() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"sku_tier": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  string(publicipprefixes.PublicIPPrefixSkuTierRegional),
```

Comment:
Just confirming here, but Is this the default that existing resources created by Terraform would have?

Reply:
Yes, any resource created has that value.
```
"sku": {
    "name": "Standard",
    "tier": "Regional"
}
```